src/random.h

```diff
@@ -89,6 +90,16 @@ class FastRandomContext {
         }
     }
 
+    uint64_t randrange(uint64_t range)
```

Comment:
This needs a comment that points out that range returned will be [0..range) and that range _must not_ be zero.